recipe/meta.yaml

```diff
@@ -78,14 +69,16 @@ test:
 
 about:
   # HTTPS does not work.
-  home: http://www.libtiff.org/
+  home: http://www.simplesystems.org/libtiff/
   license: HPND
   license_file: COPYRIGHT
   summary: Support for the Tag Image File Format (TIFF).
   description: |
     This software provides support for the Tag Image File Format (TIFF), a
     widely used format for storing image data.
-  doc_url: http://www.libtiff.org/document.html
+  # HTTPS does not work.
+  doc_url: http://www.simplesystems.org/libtiff/
+  doc_source_url: https://gitlab.com/libtiff/libtiff/-/blob/master/README.md
   dev_url: https://gitlab.com/libtiff/libtiff
```

Comment:
Unfortunately the `home` and `doc_url` only work with `http` We will have to keep it as is for now. 

Other than that everything else seems fine.